docs/scripts/helpers.js

```diff
@@ -9,11 +9,13 @@ function getLineFeed(source) {
 }
 
 const fixBabelIssuesRegExp = new RegExp(/(?<=(\/>)|,)(\r?\n){2}/g);
+const fixBabelIssuesEmptyLineImportsRegExp = new RegExp(/(import.*)\n\n(import.*)(\r?\n)/g);
```

Comment:
@eps1lon I needed to add this, otherwise, we were getting empty lines between imports, as the whole line from the `*.tsx` demo file was removed.

Reply:
It removes an empty line between imports.

Reply:
No needed in the end :)